### ISSUE TITLE
Ashes of Urh'Rok 은 어떻게 할까요?

### ISSUE THREAD

**Issue by koaTS** [dkpark89/tome4-korean-translation#196]
이쪽도 애드온으로 처리되니 번역하기가 힘들까요?

**Comment by dkpark89** [comment]
DLC는 스팀 버전에 적용되는 건가요?

**Comment by koaTS** [comment]
아니요 다운로드 버젼에도 애드온 형식으로 받을 수 있습니다. 파일은 이겁니다: https://drive.google.com/file/d/0B_pQaStwK5U5TzZ0OXM4d3c5T2c/view?usp=sharing 받으셨으면 지우겠슴당...

**Comment by dkpark89** [comment]
잘 모르겠는데.. 다음번 머징할때 이 파일들을 직접 넣어서 머징해주세요.

**Comment by KimRuin** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

alkahest7@naver.com 님께 보내신 메일 <Re: [tome4-korean-translation] Ashes of Urh'Rok DLC는 어떻게 할까요?  (#196)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by Friedsweetpotato** [comment]
초벌이라도 해두고싶은데 아직 손대기에는 이른건가요?

**Comment by koaTS** [comment]
으으.. DLC 에 있는 data/talents 들을 긁어서 번역 애드온으로 옮기고 몇개 번역해서 돌려보니까 전혀 적용이 되지 않군요. 잘 모르겠습니다..

**Comment by dkpark89** [comment]
저는 위 파일을 애드온에 넣어주면 애드온은 되는데 케릭터 생성에선 골라지지 않네요.
